code/modules/spells/spell_types/conjure.dm

```diff
@@ -65,16 +65,19 @@
 	cooldown_min = 10
 
 /obj/effect/proc_holder/spell/targeted/conjure_item/cast(list/targets, mob/user = usr)
-	if (item)
+	if (item && !qdeleted(item))
 		qdel(item)
 		item = null
 	else
 		for(var/mob/living/carbon/C in targets)
 			if(C.drop_item())
-				item = new item_type
+				item = make_item()

```

Comment:
Originally, I was going to make it have a variety of "spell packets" at random, but then decided on just one.